mlir-sycl/lib/Dialect/SYCL/IR/SYCLOps.cpp

```diff
@@ -290,6 +290,7 @@ bool SYCLUnwrapOp::areCastCompatible(TypeRange inputs, TypeRange outputs) {
   Type sourceType = inputs.front();
   Type resultType = outputs.front();
   Type bodyType = getBodyType(sourceType);
+
```

Comment:
```suggestion
```

Reply:
Yeah, I forgot to add the newline in #10139 (`SYCLWrapOp::areCastCompatible` has it) -- can I maybe sneak that change in here, please? 😇

Reply:
Allowed :smile: